server/data/applicationClient.test.ts

```diff
@@ -454,28 +454,67 @@ describeClient('ApplicationClient', provider => {
   })
 
   describe('placementApplications', () => {
-    it('calls the placement applications endpoint with the application ID', async () => {
-      const applicationId = 'applicationId'
-      const placementApplications = placementApplicationFactory.buildList(1)
+    describe('with NEW_WITHDRAWALS_FLOW_DISABLED turned on', () => {
+      it('calls the placement applications endpoint with the application ID and includeInitialRequestForPlacement=false', async () => {
+        process.env.NEW_WITHDRAWALS_FLOW_DISABLED = '1'
 
-      provider.addInteraction({
-        state: 'Server is healthy',
-        uponReceiving: 'A request for the placement applications of an application',
-        withRequest: {
-          method: 'GET',
-          path: paths.applications.placementApplications({ id: applicationId }),
-          headers: {
-            authorization: `Bearer ${token}`,
-            'X-Service-Name': 'approved-premises',
+        const applicationId = 'applicationId'
+        const placementApplications = placementApplicationFactory.buildList(1)
+
+        provider.addInteraction({
+          state: 'Server is healthy',
+          uponReceiving: 'A request for the placement applications of an application',
+          withRequest: {
+            method: 'GET',
+            path: `${paths.applications.placementApplications({ id: applicationId })}`,
+            query: {
+              includeInitialRequestForPlacement: 'false',
+            },
+            headers: {
+              authorization: `Bearer ${token}`,
+              'X-Service-Name': 'approved-premises',
+            },
           },
-        },
-        willRespondWith: {
-          status: 200,
-          body: placementApplications,
-        },
+          willRespondWith: {
+            status: 200,
+            body: placementApplications,
+          },
+        })
+
+        await applicationClient.placementApplications(applicationId)
       })
+    })
+
+    describe('with NEW_WITHDRAWALS_FLOW_DISABLED turned off', () => {
+      it('calls the placement applications endpoint with the application ID and includeInitialRequestForPlacement=true', async () => {
+        process.env.NEW_WITHDRAWALS_FLOW_DISABLED = ''
```

Comment:
Not really for this PR, but is there a reason for using `'1'` and `''` instead of `true` and `false`?